docs/workforce-management-framework/topics/con-performance-guidelines.adoc

```diff
@@ -0,0 +1,85 @@
+[id='{context}-performance-guidelines']
+= Performance Guidelines
+
+{PROJECT_NAME} integrates multiple technologies and frameworks into single solution. Developers can build on top of the provided applications and use technologies that aren't part of the standard applications.
+
+Node.js based server can rely on customer prefered database or external api and any other datasource. This  may have significant impact on overal application performance. {PROJECT_NAME} was desigined to expose areas and configurations that can have .
+Thanks to that developers can move applications to production by changing configuration.
+
+Server side application supports profiles (by default development and production) that will also help to differenciate bettween production and development configuration. This will ensure that application will be secure and perform to growing number of users.
+
+== Purpose of this guide
+
+In document provide guidelines for RainCatcher integrations by exposing common parameters and documenting performance guidelines for integrations.
+
+== Server application
+
+Server exposes number of parameters that can be changed by adjusting server configuration.
+For more information see configuration
+link:{WFM-RC-ServerURL}{WFM-RC-Release-Tag}/config-prod.js[config-prod.js]
+
+=== MongoDB
+
+MongoDB connection options should be configured to properly maintain conection pooling. See server side configuration.
+link:{WFM-RC-ServerURL}{WFM-RC-Release-Tag}/config-prod.js[config-prod.js]
```

Comment:
Maybe add a link to official mongodb documentation around what options could be tweaked.

Reply:
Had that before, but their version may differ. 
Going to include docs for the latest

Reply:
Done